.github/workflows/windows.yml

```diff
@@ -0,0 +1,42 @@
+name: Windows kDrive desktop CI
+
+on:
+  pull_request:
+    types: [synchronize, review_requested]
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-kDrive:
+    runs-on: [ self-hosted, Windows, desktop-kdrive ]
```

Comment:
As the name of the repository is `desktop-kDrive` and `kDrive` with "D" in upper-case, we may apply the same convention here.

Reply:
This section is selecting runners based on tags, so I'm using full lower-case as this is how I created the tag, I cannot 100% confirm that they are case sensitive but to avoid potential errors I'm keeping it in lower-case (the `Windows` tag was automatically created)